app/views/ReportView/components/Immune/index.tsx

```diff
@@ -52,16 +52,16 @@ const Immune = (): JSX.Element => {
         titleText="Immune Cell Types"
         canViewDetails
       />
-      {Object.values(images).length > 0 && (
+      {Boolean(images.length) && (
         <div>
           <Image
-            image={images['cibersort.combined_t-cell_scatter']}
+            image={images.find((img) => img.key === 'cibersort.combined_t-cell_scatter')}
```

Comment:
could probably shorten these to `.find(({key}) => key === 'cibersort.combined_t-cell_scatter'`